pages/_app.js

```diff
@@ -1,6 +1,7 @@
 import '../styles/globals.css'
 
 import Layout from '../components/Layout'
+const elements = ['Learn More', 'Get Involved']
```

Comment:
This probably doesn't belong here, the `_app.js` file is for setting up the layout of the Next app. Anything on this page will stay loaded between page changes, which is the main purpose of including things in here. I wouldn't export anything other the the default export. What are you trying to achieve? Can try to help you find somewhere better for this variable. It could probably go right in `index.js` where you're using it.

Reply:
Removed, using string props instead to simplify.